### PR TITLE
Feature/integrating new signup flow

### PR DESCRIPTION
**Pull request by shmbajaj**
### What is the change?
- Storing the origin url in state, to redirect the user to origin after successful login.
- If origin url is missing redirect to home.

> [Issue 463](https://github.com/Real-Dev-Squad/website-members/issues/463)

### \*Dev Tested?

- [X] Yes
- [ ] No

#### Platforms

- [X] Web

#### Browsers

- [X] Chrome

# PR on hold

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import {
+  LOGIN_URL,
+  GITHUB_MOCK_LOGIN_URL,
+  GITHUB_MOCK_LOGIN_STORAGE_KEY,
+} from '@constants/AppConstants';
+
+export function getAuthUrl() {
+  const originURL = window.location.origin;
```
Comment (ivinayakg) [inline src/utils/index.js:8]: window.location.href use this

**Review by ivinayakg — APPROVED** [review]

File: src/components/UI/navbar/index.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ const GenericClosePopUp = (ref, callback) => {
     };
   });
 };
-
+let authUrl = '';
```
Comment (ivinayakg) [inline src/components/UI/navbar/index.js:30]: change this into a useRef

**Review by ivinayakg — APPROVED** [review]
the code works, suggested one change

File: src/components/UI/navbar/index.js (hunk 2 above)
Comment (shmbajaj) [reply src/components/UI/navbar/index.js:30]: Why you want to use useRef, 
for same ref object on renders or what else?

**Review by ivinayakg — APPROVED** [review]
Thank you for making the changes. 
LGTM 👍

**Pull request merged**